mappings/net/minecraft/util/math/Vec3i.mapping

```diff
@@ -23,78 +37,148 @@ CLASS net/minecraft/class_2382 net/minecraft/util/math/Vec3i
 		ARG 1 vec
 	METHOD method_10260 getZ ()I
 	METHOD method_10262 getSquaredDistance (Lnet/minecraft/class_2382;)D
+		COMMENT {@return the squared distance between here and {@code vec}}
+		COMMENT
+		COMMENT @apiNote Passing {@link BlockPos} to this method will return a wrong value. Use
+		COMMENT {@link #getSquaredDistance(Vec3i, boolean)} with {@code treatAsBlockPos} set to
+		COMMENT {@code false} instead.
 		ARG 1 vec
 	METHOD method_10263 getX ()I
 	METHOD method_10264 getY ()I
 	METHOD method_10268 getSquaredDistance (DDDZ)D
+		COMMENT {@return the squared distance between here and {@code (x, y, z)}}
```

Comment:
```suggestion
		COMMENT {@return the squared distance between here (center or vertex) and {@code (x, y, z)}}
		COMMENT If {@code center} is {@code true}, this is equivalent to {@link Vec3d#squaredDistanceTo(double,
		COMMENT double, double) Vec3d.ofCenter(this).sqauredDistanceTo(x, y, z)}; otherwise,
		COMMENT this is equivalent to {@code Vec3d.of(this).sqauredDistanceTo(x, y, z)}.
```